src/cpp/rtps/xmlparser/XMLDynamicParser.cpp

```diff
@@ -606,6 +606,10 @@ XMLP_ret XMLParser::parseXMLBitmaskDynamicType(
     }
 
     const char* name = p_root->Attribute(NAME);
+    if (!name)
```

Comment:
I would test against nullptr just for consistency with the other test in this code block.

Reply:
Changed